student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(classroom)
+    super(name, age, parent_permission)
```

Comment:
- [ ] You still need to add the other arguments here, which are `age`, `name` and `parent_permission`.

  ```
  def initialize(age, class, name = 'Unknown', parent_permission: true)
  ```